ompi/debuggers/ompi_common_dll.c

```diff
@@ -67,9 +67,9 @@ static int host_is_big_endian = 0;
     {                                                                   \
         out_name = mqs_field_offset((qh_type), #field_name);            \
         if (out_name < 0) {                                             \
-            fprintf(stderr, "WARNING: Open MPI is unable to find "      \
+            fprintf(stderr, "WARNING: " OMPI_IDENT_STRING " is unable to find "      \
                     "field " #field_name " in the " #struct_name        \
-                    " type.  This can happen can if Open MPI is built " \
+                    " type.  This can happen can if " OMPI_IDENT_STRING " is built " \
```

Comment:
Maybe correct the typo as we go:

```suggestion
                    " type.  This can happen if " OMPI_IDENT_STRING " is built " \
```